.github/workflows/ci.yml

```diff
@@ -43,3 +39,43 @@ jobs:
           GRAFANA_API_KEY: ${{ secrets.GRAFANA_API_KEY }}
         run: node ../../sign-plugin/dist/bin/run.js --rootUrls http://www.example.com --signatureType private
         working-directory: ./packages/create-plugin/generated
+
+  release:
+    runs-on: ubuntu-latest
+    needs: test
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
We should probably do this as a separate step so we only can reference the output of that instead of repeating this logic, wdyt?

Reply:
Just to clarify - you're leaning towards merging the additional release step (along with the necessary changes to similar steps in this job (e.g. `git fetch --unshallow --tags`) into the `test` job above?

Reply:
Hehe, what? 😕 

I was thinking about doing this in a separate step that we just can reference it in the `if` statement in both test and release.

`"!contains(github.event.head_commit.message, 'ci skip') || !contains(github.event.head_commit.message, 'skip ci')"`

Should it be `or` instead of `and`?

Reply:
After having a chat, it doesn't feel like it is worth it. Lets merge this bad boy :)